config-model/src/main/java/com/yahoo/vespa/model/admin/Admin.java

```diff
@@ -149,6 +149,7 @@ public void getConfig(LogdConfig.Builder builder) {
         else {
             builder.
                 logserver(new LogdConfig.Logserver.Builder().
+                        userpc(true).
```

Comment:
Not really feedback on this PR, but shouldn't it really have been `useRpc`?

Reply:
We decided to following the existing naming convention in the logd definition.